src/app/tests/suites/TestDiscovery.yaml

```diff
@@ -201,7 +201,7 @@ tests:
     # product id ?
     - label: "TXT key for Vendor ID and Product ID (VP)"
       PICS: MCORE.SC.VP_KEY
-      optional: true
+      disabled: true
```

Comment:
Maybe file an issue on re-enabling it and reference the issue in a comment here?

Reply:
Done.